### DISCUSSION TITLE
Storyblok starter site

### DISCUSSION THREAD

**Discussion by kinkwilde** [remix-run/remix#1934] [General]
Hello friendly people of the remix.run community.

I recently stumbled across remix and have been enjoying the developer experience, pair that with my recently discovered use of storyblok I wanted to create a starter kit, for my own personal use mainly, so I have put it up onto github to get some exposure of it to other remix & storyblok developers to see if the way I have implemented things are the proposed/correct way of doing things.

So could I ask anyone from the community or development team who has some spare time to take a look through and just make a comment either here or by opening an issue.

[https://github.com/black-tape-project/remix-storyblok](https://github.com/black-tape-project/remix-storyblok)

Many thanks.

**Comment by merelinguist** [comment]
Try posting in the #showcase channel of the Discord as well!
